index.html

```diff
@@ -185,14 +211,12 @@ <h2 class="h-text-white h-text-size">
                 </li>
               </ul>
             </div>
-            <div class="btn-group">
-              <button type="button" class="btn-2">See Project</button>
-            </div>
           </div>
-          <div class="card">
+
+          <div class="card four">
             <div class="block">
               <h2 class="h-text-white h-text-size">
-                Profesional Art Printing Data
+                Profesional Art Printing Data More
               </h2>
               <p class="p-text-white">
```

Comment:
- [ ] Please look at the previous reviewer's previous comment about changing the alignment of the technologies on the projects card. Try to make it match the Figma template as best you can. currently if you notice the badges are peeking out of the bottom of the project.
- [ ] Fix this for **every** project card.


Your Design | Reference Design
-|-
![image](https://user-images.githubusercontent.com/18730106/155844727-e9fef8c5-2b4a-4909-8cea-92ee8ab0deb4.png)|![image](https://user-images.githubusercontent.com/18730106/155844715-13ddcd68-0762-43ed-b068-cdcd067f9b9f.png)